enclave.go

```diff
@@ -330,19 +330,32 @@ func (e *Enclave) Stop() error {
 func startWebServers(e *Enclave) error {
 	if e.cfg.PrometheusPort > 0 {
 		elog.Printf("Starting Prometheus Web server (%s).", e.promSrv.Addr)
-		go e.promSrv.ListenAndServeTLS("", "") //nolint:errcheck
```

Comment:
So `nolint` was a bug?

Reply:
It was meant to silence the linter, which complains about unchecked error returns. Since I now decided that we should check these returns, it's fair to call this a bug.